docs/source/aryn_cloud/accessing_the_partitioning_service.rst

```diff
@@ -54,6 +54,25 @@ Helper Functions
     # Draw the detected bounding boxes on the pages. requires poppler
     images = draw_with_boxes("mydocument.pdf", data)
 
+Different Formats
++++++++++++++++++
+
+It is easy to use multiple file types with aryn-sdk:
```

Comment:
It is easy to process files with different formats using the aryn-sdk:

Reply:
Done.